templates/accounts/registration.html

```diff
@@ -7,6 +7,23 @@
     $(function () {
     	$("#id_username").focus()
     });
+    $(document).ready(function(){
+      if (!($('#username_check')).length) {
+        $('#id_username').parent().append('<div id="username_check" style="color: #bd2d33;font-style: italic;"></div>');
```

Comment:
I changed the code to use toggle and is much better now. 
I still have to add the div with JS since I want to place it next to the username input and the hole form it's rendered using {{form.as_p}}.